nab/detectors/context_ose/cad_ose.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-

```

Comment:
Please replace your license with ours. By contributing code to this repository it falls under the Numenta license. If you'd like you can include `2016, Mikhail Smirnov   smirmik@gmail.com, https://github.com/smirmik/CAD` below the end of the license.